src/main/java/org/jboss/tools/intellij/analytics/CliConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.tools.intellij.analytics;
+
+public class CliConfig {
+    public static final String CLI_AUTH = "yes \"\" | binarypath/crda auth";
+    public static final String CLI_ANALYSE = "crda analyse filepath -j -c";
+    public static final String CLI_CONFIG_FILE = System.getProperty("user.home")+".crda/config.yaml";
+
+    public static String cliBinaryPath;
+    public String getCliBinaryPath() {
+        return cliBinaryPath;
+    }
+    public void setCliBinaryPath(String cliBinaryPath) {
+        this.cliBinaryPath = cliBinaryPath;
+    }
```

Comment:
Bit weird to see class member being set from a instance member function. I guess IJ Service should be the right method to have singleton instance of transient settings/configs.